src/tox_uv/_installer.py

```diff
@@ -67,14 +68,30 @@ def default_install_command(self, conf: Config, env_name: str | None) -> Command
         return Command(cmd)
 
     def post_process_install_command(self, cmd: Command) -> Command:
+        """Returns uv or pip based on current config.
+
+        Returns:
+            A string, uv or pip currently.
+        Raises:
+            RuntimeError: if unexpected a data is found in config.
+        """
+        install_cmd = cmd.args
+        # uv pip command does not have same option named for reinstall and we
+        # want to allow users that use original 'pip' in their install_command
+        # to still be able to make use of it.
+        if len(install_cmd) < 1 or not isinstance(install_cmd[0], str):  # pragma: no cover
+            msg = f"Unable to determine install command. {install_cmd}"
+            raise RuntimeError(msg)
+        self._pkg_manager = "uv" if install_cmd[0].endswith("uv") else "pip"
```

Comment:
I'd say custom install commands are not supported with `tox-uv` 🤔 I'd rather raise an error than support it..

Reply:
I disagree: people may want a different installer for some of the envs but not the others. Is there an alternative workaround? Can I force a different installer for a specific env?

Reply:
Why, what's the use case?

Reply:
Please expand.

Reply:
I did have few isolated cases where I wanted to use pip instead of uv installer. In these cases it was because uv installer was buggy (or it could also be because the problematic package happened to install with pip and fail with uv for their own internal assumptions). I think that we should allow users to have custom installers to allow them to implement workarounds.

On the other hand, if you want I can change the rewrite logic to do nothing if the case is unable to detect either pip or uv as being inside the install_cmd.

Reply:
> I did have few isolated cases where I wanted to use pip instead of uv installer. In these cases it was because uv installer was buggy (or it could also be because the problematic package happened to install with pip and fail with uv for their own internal assumptions).

In those cases you can just set the runner to be not the uv one, no?